src/client/containers/layout.js

```diff
@@ -100,26 +114,28 @@ class Layout extends React.Component {
 						<NavDropdown
 							eventKey={1}
 							id="create-dropdown"
+							open={this.state.menuOpen}
 							title={createDropdownTitle}
+							onToggle={this.handleDropdownToggle}
 						>
-							<MenuItem href="/work/create">
+							<MenuItem href="/work/create" onSelect={this.handleDropdownClick}>
```

Comment:
You can bind the `onSelect` event callback to the parent `NavDropdown` item, exactly like you did for `onToggle`.
For the same result, it will be more efficient since you create only one event handler instead of five.